src/components/datepicker/Datepicker.vue

```diff
@@ -12,10 +22,53 @@ export default {
     props: {
         value: Date,
         active: {
-          type: Boolean,
-          default: false
+            type: Boolean,
+            default: false
         },
-        iconPack: String
+        iconPack: String,
```

Comment:
Would you suggest just using `config.defaultIconPack` as found on line 74 of this file?